dev_scripts/template_pyxis.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+from __future__ import print_function
+import os
+import sys
+from pprint import pprint
+
+PYXIS_SERVER_URL = "https://pyxis.stage.engineering.redhat.com/"
+
+# Allow imports from parent directory.
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+
+os.environ["FRESHMAKER_DEVELOPER_ENV"] = "1"
+os.environ["FRESHMAKER_CONFIG_FILE"] = os.path.join(sys.path[0], "config.py")
+
+from freshmaker import conf, app
+from freshmaker.pyxis import Pyxis
+
+app_context = app.app_context()
+app_context.__enter__()
+
+if len(sys.argv) != 1:
+    print("Template for testing freshmaker.Pyxis class")
+    print("Usage: ./pyxis.py")
+    sys.exit(1)
+
+pyxis = Pyxis(PYXIS_SERVER_URL)
+
+def get_bundles_per_channel_per_index_image():
```

Comment:
Why isn't this called in this script?

Reply:
It's just script for testing, and even more it's template for your scripts, so it's supposed that you will change the script before using it, but it will still work and do something even without any changes.